cookbook/integrations/kubernetes/k8s_spark/pyspark_pi.py

```diff
@@ -41,6 +41,34 @@ def hello_spark(partitions: int) -> float:
            # Regular Pypsark code
            ...
 
+#. To run a Spark job on Databricks platform, just add Databricks config to the task config. Databricks Config is is the same as the databricks job request. Refer to `Databricks job request <https://docs.databricks.com/dev-tools/api/2.0/jobs.html#request-structure>`__
+
+   .. code-block:: python
+
+       @task(
+           task_config=Spark(
+               # this configuration is applied to the spark cluster
+               spark_conf={
```

Comment:
Can you `spark_conf` from the databricks example since the node types are set in the `databricks_conf`?

Reply:
it can be set in the `databricks_conf` as well. If you set it outside, it will override the spark config in databricks job.